MimusTests/MatcherTests.swift

```diff
@@ -120,6 +120,24 @@ class FoundationMatcherTests: XCTestCase {
         XCTAssertFalse(result, "Expected incorrect not to match")
     }
 
+    // MARK: Url
+
+    func testUrlPassingInvocation() {
+        let result = matcher.match(
+                expected: [URL(string: "https://fixture.url.com/fixture/suffix")!],
+                actual: [URL(string: "https://fixture.url.com/fixture/suffix")!]
+        )
+        XCTAssertTrue(result, "Expected urls to match")
+    }
+
+    func testUrlFailingInvocation() {
```

Comment:
Could we also test for what happens when we match `NSURL` against `URL` and the other way around (may be one test)?